engine/api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -42,9 +43,9 @@ case object MandatoryParameterValidator extends ParameterValidator {
 
 case object NotBlankParameterValidator extends ParameterValidator {
 
-  private final val BlankStringLiteralPattern: Pattern = stringLiteralPattern("\\s*")
+  private def stringLiteralPattern(pattern: String): Pattern = Pattern.compile(s"'$pattern'")
```

Comment:
Do we really need to wrap method Pattern.compile into another method ?

Reply:
Hmm, changed.